301-hdinsight-windows-storm-iothub/README.md

```diff
@@ -0,0 +1,22 @@
+# Install Hue as an HDInsight Iaas Cluster Application
+
+Installs Hue as an Iaas Cluster Application on an existing cluster -<br>
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fhdinsight-linux-hue-existing-cluster%2Fazuredeploy.json" target="_blank">
```

Comment:
This README doesn't seem to match the folder name (neither the description nor the link)